### PR TITLE
step-by-step guide for setting up Postgresql for portal

### LINKED ISSUE #389: step-by-step guide for setting up Postgresql for portal
## Description
As a (new) developer/user,
I needto have access to a README.md page that helps me step-by-step to setup postgresql,
so that I can proceed with Portal setup. We cannot rely on third-party Website as they could be down any moment also they might not be very welcoming to new developers. It is better than one of the new GSoC mentees that just started take this task so that they can document all the steps they take for setting up the Postgresql!

### Update [Required]
- [ ] [A separate PSQL_README.md page which shows the step-by-step guide on installing and configuring PSQL on both Windows and Ubuntu ]
### Enhancement to Update [Optional]
- [ ] [Additionally trying to do the same on CentOS]

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Two of the mentors follow the step-by-step guide and make sure it doesn't miss anything. One mentor could take the Windows tutorial and one mentor could take the Linux one.

## Estimation
[INSERT NUMBER HERE] 3 hours max for each OS.

### PR DESCRIPTION
**Pull request by ilyasofficial1617**
### Description
moved all the PSQL setup guide to a new file.
and add guide from external site.

Fixes #389 

### Type of Change:
- Documentation

### CODE REVIEW

File: PSQL_README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# How to Install PSQL
+## linux
```
Comment (tonythomas01) [inline PSQL_README.md]: this works only on Debian and its children releases. `linux` is kind of wrong usage.

File: PSQL_README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Create `systersdb` database, where `systersdb` might be any suitable name.
+
+```bash
+sudo -i -u postgres
+createuser <anyname e.g. alice> --pwprompt
+psql
+CREATE DATABASE systersdb;
+\c systersdb;
+GRANT ALL PRIVILEGES ON DATABASE systersdb to <the name>;
```
Comment (tonythomas01) [inline PSQL_README.md]: tbh - its always better to make the instructions use a standard `systersdb` so somebody copy pasting directly dont end up in trouble.

File: PSQL_README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    https://www.enterprisedb.com/downloads/postgres-postgresql-downloads
+and follow the installation process, if asked fill `5432` in port configuration
+
+# How to Configure the Database
+## Debian / Ubuntu
+Create `systersdb` database, where `systersdb` might be any suitable name.
+
+```bash
+sudo -i -u postgres
+createuser systersdb --pwprompt
```
Comment (tonythomas01) [inline PSQL_README.md]: Just saying, systerdb is not a very intuitive name for a user :D but its fine too though.

File: PSQL_README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+psql
+CREATE DATABASE systersdb;
+\c systersdb;
+GRANT ALL PRIVILEGES ON DATABASE systersdb to systerdb;
```
Comment (tonythomas01) [inline PSQL_README.md]: it gets a bit complicated here ^^

**Comment by Vaibhavi24** [comment]
However, pls squash your commits into a single commit. We follow a single commit format.

**Comment by sammy1997** [comment]
Closing this! 18 commits plus its 2 years old and there are merge conflicts. @ilyasofficial1617 please open a new PR if you still wanna work on it

**Pull request closed without merging**